README.md

```diff
@@ -23,23 +23,23 @@ Join our Discord to follow along with development: https://discord.gg/m45xPan
 - Video Files
   - Need to revive the old movie rendering code in AFile
 
-Downloads
+Prerequisites
 =======
+  - Original cd-rom or SS:EE assets in a `res/data` folder next to the executable
+    - Floppy disk assets are an older version that we can't load currently
 
-### Linux/Mac
-For now, compiling the source yourself is the way to go. See the Compiling / Running section below.
 
-### Windows
-We're making intermittent [releases](https://github.com/Interrupt/systemshock/releases) while setting up a CI system to automatically produce up-to-date ones.
+Downloads
+=======
+
+We have CI systems in place building [distributable packages](releases/) out of tagged commits for Linux, Mac and Windows.
```

Comment:
Depending on where you're viewing this .md file from, it might not always link to the releases page without the fully qualified URL here